src/Strategies/HTML.php

```diff
@@ -16,14 +16,19 @@ class HTML implements StrategyInterface
      * @return bool
      */
     public function verify(string $url, DomainVerifiableInterface $domainVerifiable)
+    {
+        $domainToken = $this->getMetaTag($url);
+        $verificationToken = DomainVerificationFacade::getTokenFor($url, $domainVerifiable)->token;
+        return $domainToken == $verificationToken;
+    }
+
+    public function getMetaTag($url)
```

Comment:
I'm confused with this method. Although the method's name means it will return the meta tag, it doesn't do that. The token is returned instead.